### DISCUSSION TITLE
Strange behavior of .airflowignore in nested folders

### DISCUSSION THREAD

**Discussion by TreasureMaster** [apache/airflow#37413] [Q&A] [answered]
Hi! I have several thousand DAGs. I wrote a DAG that transfers old DAGs to an archive folder (`airflow_dags_folder/archive`). I would like to use `.airflowignore` to ignore old DAGs in the archive folder, but it doesn’t work correctly! For example, I'm using a test Airflow container on another server with 12 DAGs. I run the command `airflow dags reserialize` in cli and see that 6 dags are synchronized when using the file. It' ok. But I see all 12 DAGs in the web interface! I wait a long time to let the service update the information. I'm trying to reload my browser window. But there is no effect. All 12 DAGs are present. Any of the syntaxes does not work - regex or glob. If I use the same entry (ignore all) in `.airflowignore` from the root folder of DAGs (`airflow_dags_folder/archive`), then everything works correctly! But it doesn't work in nested folders. How to solve this problem?
Airflow revision is 2.6.2.

**Comment by potiuk** [comment, marked as answer]
Airflow does not read .airflowignore in nested folders. You need to add approproate regexp in the top-level .airflowignore.

**Comment by TreasureMaster** [comment]
Thank you! I read the docs on `astronomer.io`, but I didn't read Airflow docs. My mistake.
